packages/renderer/src/lib/preferences/PreferencesKubernetesContextsRendering.svelte

```diff
@@ -95,32 +96,72 @@ async function handleDeleteContext(contextName: string) {
           {/if}
         </div>
         <div class="grow flex-column divide-gray-900 text-gray-400">
-          <div class="text-xs bg-charcoal-800 p-2 rounded-lg mt-1 grid grid-cols-6">
-            <span class="my-auto font-bold col-span-1 text-right">CLUSTER</span>
-            <span class="my-auto col-span-5 text-left pl-0.5 ml-3" aria-label="context-cluster">{context.cluster}</span>
-          </div>
+          <div class="flex flex-row">
+            {#if $kubernetesContextsState.get(context.name)}
```

Comment:
I have current context pointing to sandbox instance and I can query resources using `kubectl`. That means my context state should be reachable, but for some reason I don't see part of UI below.

Reply:
Did you enable the feature in the preferences, as the feature is disabled by default?